admiral/pkg/test/resources/admins@fake-cluster.k8s.local

```diff
@@ -1,7 +1,7 @@
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tDQpNSUlFN3pDQ0ExZWdBd0lCQWdJUU5MbUh5bU9wUy9Ib3Y2bFdyT3hvU1RBTkJna3Foa2lHOXcwQkFRc0ZBRENCDQpqekVlTUJ3R0ExVUVDaE1WYld0alpYSjBJR1JsZG1Wc2IzQnRaVzUwSUVOQk1USXdNQVlEVlFRTERDbHpjMmhoDQpjbTFoTXpSQWFXNTBkV2wwWkdWd1lqRXlaV0VnS0ZOb2NtbHlZVzBnVTJoaGNtMWhLVEU1TURjR0ExVUVBd3d3DQpiV3RqWlhKMElITnphR0Z5YldFek5FQnBiblIxYVhSa1pYQmlNVEpsWVNBb1UyaHlhWEpoYlNCVGFHRnliV0VwDQpNQjRYRFRJeE1EZ3dNakl5TXpJek9Gb1hEVE14TURnd01qSXlNekl6T0Zvd2dZOHhIakFjQmdOVkJBb1RGVzFyDQpZMlZ5ZENCa1pYWmxiRzl3YldWdWRDQkRRVEV5TURBR0ExVUVDd3dwYzNOb1lYSnRZVE0wUUdsdWRIVnBkR1JsDQpjR0l4TW1WaElDaFRhSEpwY21GdElGTm9ZWEp0WVNreE9UQTNCZ05WQkFNTU1HMXJZMlZ5ZENCemMyaGhjbTFoDQpNelJBYVc1MGRXbDBaR1Z3WWpFeVpXRWdLRk5vY21seVlXMGdVMmhoY20xaEtUQ0NBYUl3RFFZSktvWklodmNODQpBUUVCQlFBRGdnR1BBRENDQVlvQ2dnR0JBTWM1eHBwOCtHSmxCbGVTYTc0L0VnR3J4a3VZd3M0Mmd6UzRWS2xwDQpiVTNRSXpDV2psOVd4L0hhRVVqLzA3R1dtMlhOQWVoSS91b1JIS0x0aERMWWdMelRoazBMTDFqZkZCaDV3bDRtDQplaDlCcE5EUnhXcDNQaWhScW13SGUxNFArbDR1YU1aSHNveG5WSncvQS9La05pZXhMVXZMOVZqUXpacko4Y1luDQovTjZWVjNuc1dGQjF5dVlCdHBJUHo2ODJQS291cHpqcTVRcWdOUmo1dzFHM2p0MUFabjViK004YmNKMlQvUG1hDQp4UkNVa3NxTWpwdjZhVVdpdkd1UW5yL0xaVWpxWWhSU2xLYlNoOGdjRjNmZ3VSOC9ZTjB1OVM0R1lEa2JwMXc5DQpRYitwYk5CeS9pSkdkSzJOS0JsOHRvQmdESWx2OFlmSmdEbDdCbk1uM1J2NG02T0w0VHVVVjIva25JKy9xMkZsDQpTWW8rL0F0QUVtRGV2ZEIxbUV3emRId1AvR0NsT2QrV1hjSEV2TENibC9MZ2FIeC8zZWUyNmEwb1hzVlp4N21PDQpXMGxrMU1JTXVlbTdjMTh6WXdjRDlzUXBVVG4xRTcrTGlzWHpVNC9SdVB6MHFpQUp5RzRVZ3lqY0JzdVZ6UTNqDQo2WUNaWlUxb2ZLVzQ1aWF1QmpaOUNKS2ZVUUlEQVFBQm8wVXdRekFPQmdOVkhROEJBZjhFQkFNQ0FnUXdFZ1lEDQpWUjBUQVFIL0JBZ3dCZ0VCL3dJQkFEQWRCZ05WSFE0RUZnUVVacVdGS1BrY3d4cWFOejUvbDBhZzNYMWpGNzh3DQpEUVlKS29aSWh2Y05BUUVMQlFBRGdnR0JBSlYrRE5DanViZUdqejgyQ1FFMmNSbyszektDSm42T3pTaE85WnVCDQpFM3dxUFBJT1QvOTBtZm40dEJhT0R0NmovNi9mRTZad1YyWU93UVErR3plOGZORzJEMEZYVE16a3l3Q0VwcEkyDQpIWmtIZ0JPOE45VFhja0pIR1crYkFXc0lSRHc2V2pvbThQWTN1RDA1QWEzV2RYd2ROYUE1R2hTQ3ZVVEU0bVVYDQpESjdYVjVLZjlPdEhObllXOFZ3b3B0NlBhM2tpUHdQZE9BU0E1OVUvVVFhWTRwVE5TRnBTY1A1R1VEbnVKUTUwDQpXOEdmem1FNkV4Y0ROOXRMRnVYT0oyelFpTWpKMGVKQUxIckM2UUtrRXk0anpCTFY0RFN0SURBQ2dqbzdtU28zDQphWER0NnozbTVlSUp4TmIrMG51VTRnc1BRbnQwb2sxVW9VNVpES3ZRZ3FKcGZSMU9NYUlEWkVPREJ6dXpZYTE2DQp1V0pOQ3BEaU4rZGtBMXhQQ08wVUk4STRqcDBxek9rMjNzejlEVEwvNldNbldndkZRdWxJcVRWMDNHVE1MRnEyDQpFOXpLb0JqQzdURUJrczVwSlZmMkhwVkdrUitMVzVnZnBVT0ZFanFLYnBhMEorV1hHVVRSU3R1UVBiNjQzRVBTDQpZeEl6TGR4OW04RU1zeHpYTjNQUGpDVTYxdz09DQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tDQo=
```

Comment:
Just curious, why was an update required to ca data?

Reply:
somehow I was getting the following error so I updated it.
```
destinationrule_test.go:28: Unexpected err failed to create destination rule controller k8s client: unable to load root certificates: unable to parse bytes as PEM block
    destinationrule_test.go:32: DestinationRule controller should never be nil without an error thrown
--- FAIL: TestNewDestinationRuleController
```

Reply:
Cool, Just ran openssl command after decoding using base64, and the old one was not a certificate.
![Screen Shot 2022-08-17 at 11 09 45 AM](https://user-images.githubusercontent.com/5599566/185211856-7012491d-cd50-44bb-becf-22a497c45bc3.png)